### PR TITLE
Updating custom liquid section

### LINKED ISSUE #2645: Custom Liquid section copy updates
## Describe the enhancement you'd like
Updates to the Custom Liquid section's copy to address feedback from the recent no-code friction log.

## Additional context/screenshots
Current
image

Recommended updates 
image

### PR DESCRIPTION
**Pull request by lougoncharenko**
### PR Summary: 

Custom Liquid Section is updated based off of feedback from the recent no-code friction log.

### Why are these changes introduced?

Fixes #2645.

### What approach did you take?
I changed the wording in the label and info for custom liquid to make the new recommendations.

### Visual impact on existing themes

The new changes will have no applicable changes to merchants current theme versions. 

### Testing steps/scenarios

- [ ] Test custom liquid section n featured product
- [ ] Test custom liquid section in main products 

### Demo links

- [Editor](https://os2-demo.myshopify.com/admin/themes/140148113430/editor?block=template--17289451110422__9fedfed6-d55c-428c-bd8f-9439a4789ef9%2F9698217d-1746-446c-b184-0c7113920559&section=template--17289451110422__9fedfed6-d55c-428c-bd8f-9439a4789ef9)

### Checklist
- [x] Added PR summary for [release notes](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)
- [x] Requested review from UX (Only for changes that are affecting the experience or perceivable visual details)
- [x] Created a ticket for the [help.shopify.com](https://help.shopify.com) documentation team about updates to theme settings. (Internal-only task)
- [x] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [x] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [x] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [x] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [x] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

**Review by ludoboludo — APPROVED** [review]

**Comment by katycobb** [comment]
Looks good to me, thanks!

File: locales/en.default.schema.json [hunk 1]
```diff
@@ -651,8 +651,8 @@
       "name": "Custom Liquid",
       "settings": {
         "custom_liquid": {
-          "label": "Custom Liquid",
-          "info": "Add app snippets or other Liquid code to create advanced customizations."
+          "label": "Liquid Code",
```
[the remark this answers is not in the document]
Comment (Precha24) [reply locales/en.default.schema.json:654]: > **Localization quality issue found**
> 
> The following issues may affect the quality of localized translations if they are not addressed:
> 
> * The value `Liquid Code` for key `sections.custom-liquid.settings.custom_liquid.label` is very short. Short strings are more likely to be misunderstood by translators without context. Please provide [additional context](https://vault.shopify.io/page/Providing-context-for-translators~dhb537f.md) for the translators if possible.
> 
> Questions about these messages? Hop in the [#help-localization](https://shopify.slack.com/messages/C7TJQLVC7) Slack channel.
>

[the remark this answers is not in the document]
Comment (lougoncharenko) [reply locales/en.default.schema.json:654]: Screenshot 2023-06-21 at 1 08 23 PM

[the remark this answers is not in the document]
Comment (lougoncharenko) [reply locales/en.default.schema.json]: Screenshot 2023-06-22 at 8 12 31 AM

**Review by kmeleta — APPROVED** [review]

**Pull request merged**